packages/tizen_app_manager/lib/src/app_context.dart

```diff
@@ -41,204 +17,118 @@ class _AppManager {
   _AppManager() {
     final DynamicLibrary libAppMananger =
         DynamicLibrary.open('libcapi-appfw-app-manager.so.0');
-    getAppContext = libAppMananger
-        .lookup<NativeFunction<_AppManagerGetAppContext>>(
-            'app_manager_get_app_context')
-        .asFunction();
-    appIsRunning = libAppMananger
-        .lookup<NativeFunction<_AppManagerIsRunning>>('app_manager_is_running')
-        .asFunction();
-    getPackageId = libAppMananger
-        .lookup<NativeFunction<_AppContextGetPackageId>>(
-            'app_context_get_package_id')
-        .asFunction();
-    getProcessId = libAppMananger
-        .lookup<NativeFunction<_AppContextGetPid>>('app_context_get_pid')
-        .asFunction();
-    getAppState = libAppMananger
-        .lookup<NativeFunction<_AppContextGetAppState>>(
-            'app_context_get_app_state')
-        .asFunction();
+    // Since app_manager_terminate_app is non public API, load it using ffi directly.
```

Comment:
@WonyoungChoi I can't get app_manager_terminate_app from tizen_interop, so I had to leave it as is.